classes/Langchecker/Project.php

```diff
@@ -224,6 +224,40 @@ public static function getWebsiteDataType($website)
     }
 
     /**
+     * Return localized URL for stage (if available)
+     *
+     * @param   array   $reference_data  Data for reference locale
+     * @param   string  $locales         Requested locale, empty to get the
+     *                                   generic URL without locale code
+     * @param   string  $type            If the return value is a plain URL or
+     *                                   HTML link
+     *
+     * @return  string                   Localized URL
+     */
+    public static function getLocalizedURL($reference_data, $locale = '', $type = 'txt')
+    {
+        if (! isset($reference_data['url'])) {
+            // No URL available for this page
+
+            return '-';
+        }
+
+        $page_url = $reference_data['url'];
+        if ($locale == '') {
+            // No locale provided, need a neuter URL
+            $page_url = str_replace('/%LOCALE%', '', $page_url);
+        } else {
+            $page_url = str_replace('/%LOCALE%', "/{$locale}", $page_url);
```

Comment:
Interesting, I never used arrays in str_replace. I'd go for dropping the line above too with the first assignment to ```$page_url``` and
```
        $page_url = str_replace(
                        ['%LOCALE%', '//', ':/'], 
                        [$locale, '/', '://'], 
                        $reference_data['url']
                    );
```